tests/queryset/test_queryset.py

```diff
@@ -6006,6 +6006,14 @@ def test_allow_disk_use(self):
         for index in range(qs_disk.count()):
             assert qs_disk[index] == qs[index]
 
+    def test_filter_on_empty_list_of_floats(self):
+        class Family(Document):
+            ages = ListField(field=FloatField())
+
+        Family(ages = [7.5, 5.5, 29.5, 32.7]).save()
+        Family(ages = []).save()
+        assert Family.objects(ages__gt=[]).count() == 1
```

Comment:
Further to the comment made in the associated Issue, which one are you expecting to match here? [] or [7.5, 5.5, 29.5, 32.7] ? I believe it's [7.5, 5.5, 29.5, 32.7] but the test case is ambiguous